src/xdebugger.cpp

```diff
@@ -287,9 +287,20 @@ namespace xpyt
             py::module xeus_python_shell = py::module::import("xeus_python_shell.debugger");
             m_pydebugger = xeus_python_shell.attr("XDebugger")();
 
+            // Get debugpy version
             std::string expression = "debugpy.__version__";
             std::string version = (eval(py::str(expression))).cast<std::string>();
-            m_copy_to_globals_available = less_than_version(version, "1.6.5");
+
+            // Format the version to match [0-9]+(\s[0-9]+)*
+            int pos = version.find_first_of("abrc");
```

Comment:
`find_first_pos` returns an unsigned int, so this should be `size_t pos = ...` . Then pos must be compared to `std::string::npos`: `if (pos != std::string::npos) ... `

Reply:
Thanks, I first tried it but I was comparing it to `-1` and the compiler was not OK :smile: